README.md

```diff
@@ -101,15 +101,25 @@ If these cfg files are not present, create them as they will be used to when lau
 
 #### Running the Server
 
-A server can be run as follows
+First, make sure the server target has been built
```

Comment:
As this is in the running the server section, this information should probably migrate to ### Step 3. Configure and build

Otherwise we are starting to cross pollinate a little.

Reply:
tbh I waivered between this and putting it there; I guessed maybe its omission meant that it's more situational when you'd need to build & run the server (vs. the client/GameLauncher)? But if most folks are going to want to run & build both, then agreed Step 3 makes more sense.